ReadDG12.md

```diff
@@ -0,0 +1 @@
+ReadDG12.md 
```

Comment:
Is this file intentionally included?

Reply:
I intended this as a place holder unless you prefer it not be included or can you suggest another name for a secondary readme